extensions/github-authentication/src/githubServer.ts

```diff
@@ -578,6 +579,14 @@ export class GitHubEnterpriseServer implements IGitHubServer {
 
 	private getServerUri(path: string = '') {
 		const apiUri = vscode.Uri.parse(vscode.workspace.getConfiguration('github-enterprise').get<string>('uri') || '', true);
+		const parsedPublicGithubUri = vscode.Uri.parse(GITHUB_PUBLIC_API);
```

Comment:
You're really only using the authority down below so instead, can you make the constant above `GITHUB_PUBLIC_AUTHORITY` and have that be just `api.github.com` (in other words, just the authority) and do the if with that string?